ros/src/computing/perception/detection/object_filter/packages/lidar_shape_estimation/lidar_shape_estimation/src/model/convex_hull.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * v1.0 Yukihiro Saito
+ */
+
+#include "convex_hull.hpp"
+
+bool ConvexHullModel::estimate(const pcl::PointCloud<pcl::PointXYZ>& cluster, autoware_msgs::DetectedObject& output)
```

Comment:
@yukkysaito is this function a placeholder for something else in the future?

Reply:
Indeed. The idea is to support in the future different “shapes” to estimate their pose.

Reply:
OK, might be worth adding a comment with a brief explanation for clarity

Reply:
@amc-nu thank you.https://github.com/CPFL/Autoware/pull/1815/commits/c6da6036a2a8c1159a3892579fdb556fac39fee9